sos/report/plugins/cxl.py

```diff
@@ -0,0 +1,109 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class cxl(Plugin, IndependentPlugin):
+    """This plugin collects data from Compute Express Link (CXL) devices
+    """
+
+    short_desc = 'Compute Express Link (CXL)'
+    plugin_name = 'cxl'
+    profiles = ('storage', 'hardware', 'memory')
+    # Utilities can be installed by package or self compiled
+    packages = ('cxl-cli', 'daxctl')
+    commands = ('cxl', 'daxctl')
+
+    def setup(self):
+        """ Use the daxctl-list(1) command to collect:
+        -i, --idle
+           Include idle (not enabled / zero-sized) devices in the listing
+
```

Comment:
I don't think we need to carry `--help` output in comments in the plugin, especially for options we don't end up using.

Reply:
Thanks for the feedback. The args will be used in the next revision of the plugin as the OS and Hardware availability mature. I can remove them now and re-add them in the near future or keep them. Let me know.

Reply:
Help info was removed and replaced with a simpler message.